src/syntax/unicode.mli

```diff
@@ -1,3 +1,5 @@
+val version : string
```

Comment:
It might be nice to have a comment noting that the version is autogenerated from the Unicode database files.